server/blob/blob-objectstorage/src/main/java/org/apache/james/blob/objectstorage/ObjectStorageBlobsDAO.java

```diff
@@ -185,4 +189,14 @@ public BucketName getDefaultBucketName() {
     public PayloadCodec getPayloadCodec() {
         return payloadCodec;
     }
+
+    @VisibleForTesting
+    public Mono<Void> deleteAllBuckets() {
```

Comment:
you really want to declare such a public API?

Reply:
See https://github.com/linagora/james-project/pull/2517 : it will become package private by using prefix as an isolation strategy in Guice integretion tests.

I hesitated adding that comit here but prefered keeping the diffs small